src/main/java/org/wildfly/bot/util/GitHubBotNameProvider.java

```diff
@@ -0,0 +1,57 @@
+package org.wildfly.bot.util; // Or a suitable package like org.wildfly.bot.config or org.wildfly.bot.github
+
+import io.quarkiverse.githubapp.GitHubClientProvider;
+import jakarta.annotation.PostConstruct;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import org.jboss.logging.Logger;
+
+import java.io.IOException;
+
+/**
+ * Provides the dynamically fetched name of the GitHub Bot.
+ * The name is fetched once during application startup.
+ */
+@ApplicationScoped
+public class GitHubBotNameProvider {
+
+    private static final Logger LOG = Logger.getLogger(GitHubBotNameProvider.class);
```

Comment:
```suggestion
    private static final Logger logger = Logger.getLogger(GitHubBotNameProvider.class);
```
as we previously talked about, unless you want to leave it as (for the sake of consistency) it is and fix it later.